modules/microshift-ki-cni-iptables-deleted.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * microshift_troubleshooting/microshift-known-issues.adoc
+
```

Comment:
This issue (firewall reload requires ovnkube-master restart) has been fixed in both 4.13 and latest 4.12, I think we can remove this chapter if no other known issues need to be documented.
4.13 fix: https://github.com/openshift/microshift/pull/1300
4.12 fix: https://github.com/openshift/microshift/pull/1309

Reply:
I made a separate [Jira](https://issues.redhat.com/browse/OSDOCS-5318) for this task. There is other work related to it from a docs standpoint that I need to do.